source/clear-linux/get-started/get-started.rst

```diff
@@ -1,7 +1,7 @@
-.. _getting-started:
+.. _get-started:
```

Comment:
Changing this label means that all \:ref\:\`getting-started\` instances must also be changed.